src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,88 @@
+package seedu.addressbook.ui;
+
+import java.io.PrintStream;
+import java.util.List;
+
+/**
+ * Does all formatting and styling of output text for the application.
+ */
+
+public class Formatter {
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+    
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+
+    private final PrintStream _out;
```

Comment:
This naming style (using `_` as prefix/suffix for local variables), although allowed by our coding standard, is not consistent with the rest of the code base.